docs/04 Use cases/04_cw20/04_cw20.md

```diff
@@ -1,19 +1,19 @@
-# CW20
+# CW20 - Fungible Tokens with CosmWasm
 
-The `wasm` adaptation of the ERC20 contract is called the CW20, it encapsulates the same logic as the ERC20|ERC777 contract and allows for token creation on the a blockchain with the `wasm` module. 
+The `wasm` adaptation of the ERC20 contract is called the CW20, it encapsulates the same logic as the ERC20|ERC777 contract and allows for token creation on the blockchain with the `wasm` module. 
 
 ## Create a smart contract using  Rust and WebAssembly on the Cosmos SDK
 
-As mentioned in the previous chapter, the `wasm` module allows to use WebAssembly code (WASM) on a Cosmos SDK blockchain. With starport, we can add the module very conviniently with
+As mentioned in the previous chapter, the `wasm` module allows using WebAssembly code (Wasm) on a Cosmos SDK blockchain. With starport, we can add the module very conveniently with
 
 ```bash
 starport module import wasm
 ```
 
 ## Upload the contract to a starport chain
 
-The `wasm` module does not use solidity but rust as language, therefore it is necessary to have rust installed on the machine you are plan to compile and upload the smart contract. In the following steps we will download the ERC20 code, test it, deploy it for production, host it on a the blockchain and use it.
-The requirements are:
+The `wasm` module does not use solidity but rust as language, therefore it is necessary to have rust installed on the machine you are plan to compile and upload the smart contract. In the following steps, we will download the CW20 code, test it, deploy it for production, host it on a blockchain and use it.
```

Comment:
```suggestion
The `wasm` module uses Rust language instead of Solidity, therefore it is necessary to have Rust installed on the machine you are planning to compile and upload the smart contract. In the following steps, we will download the CW20 code, test it, deploy it for production, host it on a blockchain and use it.
```